tests/test_tod_ops.py

```diff
@@ -121,6 +153,28 @@ def test_basic(self):
                 assert_allclose(tod.signal[1][gap_mask], sentinel)
                 # ... check "extraction" has model values.
                 assert_allclose(ex[1].data, sig[gap_mask], atol=atol)
+    
+    def test_fillglitches(self):
+        """Tests fill glitches wrapper function"""
+        ts = np.arange(0, 1*60, 1/200)
+        aman = get_glitchy_tod(ts, ndets=100)
+        # test poly fill
+        up, mg = False, False
+        glitch_filled = tod_ops.gapfill.fill_glitches(aman, use_pca=up,
+                                                      wrap=mg)
+        self.assertTrue(np.max(np.abs(glitch_filled-aman.inputsignal)) < 1e-3)
+
+        # test pca fill
+        up, mg = True, False
+        glitch_filled = tod_ops.gapfill.fill_glitches(aman, use_pca=up,
+                                                      wrap=mg)
+        print(np.max(np.abs(glitch_filled-aman.inputsignal)))
+
+        # test in_place
```

Comment:
This isn't really "in_place".  "in_place" would mean that tod.signal is modified directly, without creating a whole new vector for glitch_filled.  Also, there's no harm in return glitch_filled if it was also wrapped.

Reply:
Do you want me to make an in_place option or just change the comment to "test wrap new field"? Would you prefer I always return glitch_filled so instead of checking if the returned value is None here instead check if the new field exists in the axis manager?

Reply:
Just change the comment.

Yes, I think it's better to always return glitch_filled, and instead check that the correct new field was created in the axisman.